File: 0724-find-pivot-index/0724-find-pivot-index.c
int pivotIndex(int* nums, int numsSize){
    int sum = 0;
    for (int i = 0; i < numsSize; i++)
    {
        sum += nums[i];
    }
    int ls = 0;
    for (int i = 0; i < numsSize; i++)
    {
        if (ls == (sum - ls - nums[i]))
        {
            return i;
        }
        ls += nums[i];
    }
    return -1;
}

File: 0724-find-pivot-index/0724-find-pivot-index.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var pivotIndex = function(nums) {
    const sum = nums.reduce((c, n) => c + n, 0);
    let ls = 0;
    for (let i = 0; i < nums.length; i++) {
        if (ls == (sum - ls - nums[i])) {
            return i;
        }
        ls += nums[i];
    }
    return -1;
};